app/src/main/java/org/mozilla/fenix/onboarding/view/Onboarding.kt

```diff
@@ -164,7 +164,11 @@ private fun OnboardingWelcomeBottomContent(
 @Composable
 private fun OnboardingWelcomeContent() {
     Column(
-        modifier = Modifier.padding(horizontal = 16.dp),
+        modifier = Modifier.padding(
+            start = 16.dp,
+            end = 16.dp,
+            bottom = 32.dp,
```

Comment:
Because this is adding padding only to the bottom it will still be considered in the arrangement. To center this in the space you would need to add a min padding of 32.dp to the top as well (assuming that is ok with design). So final result would be:
`Modifier.padding(horizontal = 16.dp, vertical = 32.dp)`

Reply:
This makes sense to me.